pkg/peerprotocol/fullnode.go

```diff
@@ -20,3 +20,9 @@ func NewFullNodeProtocol(connection *Connection) (*FullNodeProtocol, error) {
 func (c *FullNodeProtocol) RequestPeers() error {
 	return c.connection.Do(protocols.ProtocolMessageTypeRequestPeers, &protocols.RequestPeers{})
 }
+
+// RequestBlock asks the current peer to respond with a block
+func (c *FullNodeProtocol) RequestBlock(heigth uint32, includeTransactionBlock bool) error {
```

Comment:
This is somewhat of a stylistic thing we settled on for this library, but if you look at the RPC package, we usually pass an `opts` struct to functions like this, instead of individual function params, so that when/if the calls have lots of options, its easy to update in the future without totally changing the function signature.

See this for an example (and below for how its passed to the RPCs) https://github.com/Chia-Network/go-chia-libs/blob/main/pkg/rpc/fullnode.go#L94

You already defined the new type for `RequestBlock` anyways, so maybe just make that the function args here?

Reply:
Yes, it's more future-proofed. 👍🏻